exist-core/src/main/java/org/exist/storage/btree/Repair.java

```diff
@@ -56,22 +56,22 @@ public void repair(String id) {
         try(final DBBroker broker = pool.get(Optional.of(pool.getSecurityManager().getSystemSubject()))) {
 
             BTree btree = null;
```

Comment:
I would try to have the switch statement return the value that is then assigned to btree
```java
btree = switch (id) { … }

Reply:
If we are lucky btree can be final then